packages/frontend/app/(dashboard)/course/[cid]/(settings)/settings/components/CourseRosterTable.tsx

```diff
@@ -64,15 +65,16 @@ const CourseRosterTable: React.FC<CourseRosterTableProps> = ({
   const fetchUsers = async () => {
     const data = await API.course.getUserInfo(courseId, page, role, search)
     setUsers(data.users)
+    setTotalUsers(data.total)
   }
 
   useEffect(() => {
-    fetchUsers()
+    fetchUsers().then()
```

Comment:
empty `.then()`s?

Reply:
has no effect besides removing an irritating warning annotation when an async method is called without 'await' or '.then()'